src/frontend/src/binder/expr/function/aggregate.rs

```diff
@@ -139,12 +139,9 @@ impl Binder {
         let order_by = OrderBy::new(vec![self.bind_order_by_expr(within_group)?]);
 
         // check signature and do implicit cast
-        match (&kind, direct_args.as_mut_slice(), args.as_mut_slice()) {
-            (
-                AggKind::Builtin(PbAggKind::PercentileCont | PbAggKind::PercentileDisc),
-                [fraction],
-                [arg],
-            ) => {
+        match (&kind, direct_args.len(), args.as_mut_slice()) {
```

Comment:
Match on `len` instead of `mutable slice`. This is because we want to push a `direct_arg` of `relative_error` into the `direct_args` vec. A `mutable slice` doesn't allow us to do that, and we can't match on a vector's contents.

So we just match on the len instead.